### PR TITLE
Bug/EWL-4326 grids broken

### PR DESCRIPTION
**Pull request by designerneil**
## Ticket(s)

Please do not submit a Pull Request without a relevant ticket! If you are creating a new pattern or feature, create an issue describing the need for this feature Github **first** and then link to it below.

**Jira Ticket**

- [EWL-EWL-4326:  Grids broken in IE11](https://issues.ama-assn.org/browse/EWL-4326)

## Description

The CSS grid breaks in IE11

## To Test
- [ ] Switch to this branch
- [ ] Go to browserstack: http://ama-d8.local/test/topic_1

## Relevant Screenshots/GIFs
screen shot 2017-12-06 at 9 15 59 am

## Remaining Tasks

none

## Additional Notes

none

### CODE REVIEW

File: styleguide/source/_patterns/03-templates/04-topic/topic.twig [hunk 1]
```diff
@@ -5,13 +5,9 @@
...
 
-  <div class="grid grid-margin">
-    <div class="topic_top grid-region">
-      {% include 'atoms-page-title' %}
-    </div>
-  </div>
 
-  <div class="topic_main grid-margin">
+
+  <div class="topic_main grid-margin css-grid">
```
Comment (froboy) [inline styleguide/source/_patterns/03-templates/04-topic/topic.twig]: If this is going to require new styles you'll need to either open a corresponding PR on D8 or create a ticket for someone else to do that. Please link either the PR or the ticket before this one is merged.

File: styleguide/source/_patterns/03-templates/04-topic/topic.twig [hunk 2]
```diff
@@ -5,13 +5,9 @@
 
 <main class="topic container">
 
-  <div class="grid grid-margin">
```
Comment (froboy) [inline styleguide/source/_patterns/03-templates/04-topic/topic.twig]: Why is the title being removed?

Comment (designerneil) [reply styleguide/source/_patterns/03-templates/04-topic/topic.twig]: the title section has been restored. good catch

**Comment by froboy** [comment]
IE11 WIndows 8 looks great!
dashboard

just for kicks I tested on IE10 and while the rest of the page has some issues, the grid still seems to work too!
dashboard

Testing the D8 ticket now.

**Comment by froboy** [comment]
Couple things I found in D8 testing:

- Something strange happens when there's a youtube video in the page. It seems to make the center column take up more space than it should.
pattern_lab_-_templates-topic_and_pattern_lab_-_templates-topic

- Extra padding is introduced between the page title and content. This should be fixed.
pattern_lab_-_templates-topic_and_pattern_lab_-_templates-topic

File: styleguide/source/_patterns/03-templates/04-topic/_topic.scss [hunk 3]
```diff
@@ -3,40 +3,53 @@
   display: block;
 }
 
-.topic .topic_main {
+.topic .topic_main.css-grid {
   display: grid;
+  display: -ms-grid;
   grid-template-columns: 100%;
-  grid-column-gap: 0;
+  padding: 28px 0;
```
Comment (froboy) [inline styleguide/source/_patterns/03-templates/04-topic/_topic.scss]: This seems to behave differently than the `grid-column-gap` property. See screenshot.

**Comment by designerneil** [comment]
@froboy apparently I forgot to add 
-ms-grid-columns: 1fr 2fr 1fr;
To the grid container

**Comment by froboy** [comment]
pattern_lab_-_templates-topic_and_pattern_lab_-_templates-topic

still seeing the title spacing issue.

**Comment by designerneil** [comment]
@froboy i don't see the spacing issue. But just in case I've pushed a change which removes all padding. Do you still weirdness with the video stretching center column?

**Comment by froboy** [comment]
dashboard
pattern_lab_-_templates-topic_and_pattern_lab_-_templates-topic

video looks good and spacing looks good. thanks @designerneil!

**Pull request merged**